apps/dotcom/src/components/IFrameProtector.tsx

```diff
@@ -26,8 +26,28 @@ and we should show an annoying messsage.
 If we're not in an iframe, we don't need to do anything.
 */
 
+export enum ROOM_CONTEXT {
```

Comment:
We hardly ever use enums in the repo, we seem to prefer unions. Feels kinda weird to see it 😄

Reply:
yeah, unions are super hard to grep for in the codebase. especially with generic words like, say, `'room'` this makes it hard to track down where things are being used. it's all about refactorability!